gtda/images/filtrations.py

```diff
@@ -1036,3 +1038,227 @@ def plot(Xt, sample=0, colorscale='greys', origin='upper',
             title=f"Signed-distance filtration of image {sample}",
             plotly_params=plotly_params
             )
+
+
+@adapt_fit_transform_docs
+class DensityFiltration(BaseEstimator, TransformerMixin, PlotterMixin):
+    """Filtrations of 2D/3D binary images based on the number of neighboring
+    activated pixels.
+
+    The density filtration assigns to each pixel of a binary image a greyscale
+    value equal to the number of activated pixels with a ball centered around
+    it.
+
+    Parameters
+    ----------
+    radius : float, default: ``1``
```

Comment:
Maybe ``1.`` to stress the floatness? I think we do this elsewhere when params are float. Also we would normally go "float, optional, default: [...]"

Reply:
Yep, indeed, I just forgot it here!